pkg/k8s/k8s.go

```diff
@@ -15,20 +15,28 @@ const (
 	ReplicationControllers = "replicationcontrollers"
 	Services               = "services"
 	All                    = "all"
+	Authority              = "authorities"
```

Comment:
Ah, sorry! I was unclear here. I think we should also change the const name from `Authority` to `Authorities`.

Reply:
oh yeah we totally should. that slipped by me too.